docs/README.md

```diff
@@ -72,38 +72,38 @@ built from self-describing data structures.  Everything in a Zed lake
 is built from Zed data and each system component can be run and tested in isolation.
 
 Since Zed data is self-describing, this approach makes stream composition
-very easy.  Data from a Zed query can be trivially be piped to a local
+very easy.  Data from a Zed query can trivially be piped to a local
 instance of `zq` by feeding the resulting Zed stream to stdin of `zq`, for example,
 ```
-zed query "from pool | ... remote query..." | zq "...local query..." -
+zed query "from pool | ...remote query..." | zq "...local query..." -
 ```
 There is no need to configure the Zed entities with schema information
 like [proto configs](https://developers.google.com/protocol-buffers/docs/proto3)
 or connections to
 [schema registries](https://docs.confluent.io/platform/current/schema-registry/index.html).
 
-A Zed lake is completely self contained requiring no auxiliary databases
+A Zed lake is completely self-contained, requiring no auxiliary databases
 (like the [Hive metastore](https://cwiki.apache.org/confluence/display/hive/design))
 or other third-party services to interpret the lake data.
-Once copied, a new service can be instantiated by pointing a `zed service`
+Once copied, a new service can be instantiated by pointing a `zed serve`
 at the copy of the lake.
 
 Functionality like indexing, data compaction, and retention are all
 API-driven.
 
 Bite-sized components are unified by the Zed data, usually in the ZNG format:
-* All lake meta-data is available via metaqueries.
+* All lake meta-data is available via meta-queries.
 * All like operations available through the service API are also available
 directly via the `zed` command.
-* Everything available to the client is
+* Everything available to the client is XXX
```

Comment:
I noticed this sentence ends unfinished. I'm not sure what was intended, so maybe someone can step in and finish it. Otherwise the bullet should probably be removed for now.

Reply:
oops, I think this was a point about everything the Brim app has access to is available in the API, but I think since this in the zed repo, let's just delete this bullet.